### DISCUSSION TITLE
Web-Interface funktioniert nur mit Internet-Verbindung

### DISCUSSION THREAD

**Discussion by thanx2you** [softwarecrash/Victron2MQTT#18] [General]
Das Web-Interface  des Victron2MQTT und Daly2MQTT funktioniert leider ohne Internet nicht richtig.
Die Scripte werden offenbar von  ajax.googleapis.com und jsdelivr.net geladen.
Gibt es eine Möglichkeit in einem isolierten Netz die ausgelesenen Werte im Webinterface trozdem anzuzeigen? 

Vielen Grüße

**Comment by softwarecrash** [comment]
Aktuell gibt es dafür keine lösung.
Allerdings sollte der Browser die assets im Cache halten und nutzen
